modules/tiles/src/types.ts

```diff
@@ -9,17 +7,17 @@ export type BoundingRectangle = {
  * We can't import it from Deck.gl to avoid circular reference */
 export type Viewport = {
   id: string;
-  cameraPosition: number[] | Vector3;
+  cameraPosition: [number,number,number] ;
```

Comment:
This won't be compatible with Vector3 (which inherits `Array<number>`) and will break code. We should probably use the `Vector3Like` type from math.gl or equivalently `[number,number,number] | Vector3`.

Also make sure to run `yarn lint fix` so that prettier autoformats the code, or insert spaces after commas.